docs/source/plotly_figures/plot_pareto_front.html

```diff
@@ -0,0 +1,20 @@
+<div>
```

Comment:
This file was added to embed the plotting result of `plot_pareto_result()` to the official API reference (c.f. https://github.com/optuna/optuna/pull/1003).